src/ios/ffi_for_sanity_tests.rs

```diff
@@ -165,3 +232,35 @@ fn register_callback_internal(callback: Box<dyn Callback>) {
         }
     });
 }
+
+fn register_log_callback_internal(callback: Box<dyn LogCallback>) {
```

Comment:
Hmm, channel may be overkill for this, but it shouldn't have downsides for now (other than performance, maybe?)

Reply:
Did you look into any alternatives?

Reply:
Not yet, but we should come back to this for future versions. We should also benchmark FFI.